matlab/test/Ice/defaultValue/Test.ice

```diff
@@ -119,36 +119,6 @@ class Derived extends Base
     Nested::Color nc3 = blue;
 }
 
-exception BaseEx
```

Comment:
We had a fairly comprehensive testing for the user exception constructors even through it makes no sense to construct a user exception from a client-side only mapping.